src/api/app/assets/stylesheets/webui/responsive_ux/_layout.scss

```diff
@@ -21,18 +23,26 @@
 
   .menu-right { cursor: pointer; }
   .menu-options { font-size: 1.1rem }
+
+  .navbar-brand img { height: 30px; }
 }
 
-#bottom-navigation { height: 58px; }
+#bottom-navigation { height: $bottom-bar-height; }
 
 #navigation, #bottom-navigation {
   a.nav-link {
+    .small {
+      line-height: 1rem;
+      padding-top: 0.25rem;
+    }
+
     &.active {
       background-color: darken($dark, 10%);
       @extend .rounded;
     }
   }
 }
+
```

Comment:
Is this intended?

Reply:
Sorry, I don't know what you mean.

Reply:
@danidoni  means the space on the line 45 @saraycp

Reply:
So yes! it's intended, I added an empty line before and after each comment.

Reply:
Ok!